pkg/test/kube/consts.go

```diff
@@ -0,0 +1,20 @@
+//  Copyright 2018 Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package kube
+
+const (
+	// KubeSystemNS is the name of the kube-system namespace
+	KubeSystemNS = "kube-system"
```

Comment:
Nothing should use this value directly. It should merely be a default for the kube setting `systemNamespace`.

Reply:
This is not istio-system. It is kube-system.

But it looks like it is not used anymore (I used to use this for Helm+Tiller based installation).

Removing the file.